neuralprophet/utils_torch.py

```diff
@@ -96,3 +97,50 @@ def create_optimizer_from_config(optimizer_name, model_parameters, lr):
     else:
         raise ValueError
     return optimizer
+
+
+def interprete_model(target_model, net, forward_func):
+    """
+    Returns model input attributions for a given network and forward function.
+
+    Parameters
+    ----------
+        net : str
+            Name of the network for which input attributions are to be computed.
+
+        forward_func : str
+            Name of the forward function for which input attributions are to be computed.
+
+    Returns
+    -------
+        torch.Tensor
+            Input attributions for the given network and forward function.
+    """
+    # Load the respective forward function from the model and init model interpreter
+    forward = getattr(target_model, forward_func)
+    saliency = Saliency(forward_func=forward)
```

Comment:
fyi - I got rid of the "InterpretableModel" here and simply pass the forward function to the attribution method